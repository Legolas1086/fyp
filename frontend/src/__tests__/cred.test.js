import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Credential from '../cred'

describe('snapshot testing',()=>{
    test("renders approve the request compnenet",()=>{
        const fragment = render(
            <BrowserRouter>
            <Credential />
            </BrowserRouter>
        ).asFragment();
    expect(fragment).toMatchSnapshot()
    })
});