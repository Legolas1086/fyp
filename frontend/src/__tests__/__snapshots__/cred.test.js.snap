// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing renders approve the request compnenet 1`] = `
<DocumentFragment>
  <div
    class="OuterContainer"
  >
    <div
      class="limiter"
    >
      <div
        class="wrap-login"
      >
        <form
          class=""
        >
          <div
            class="avatar"
          >
            <img
              src="avatar.svg"
              width="60"
            />
          </div>
          <div
            class="mb-3"
          >
            <label
              class="form-label"
              for="formBasicEmail"
            >
              Email address
            </label>
            <input
              class="form-control"
              id="formBasicEmail"
              placeholder="Enter email"
              type="email"
            />
            <small
              class="text-muted form-text"
            >
              We'll never share your email with anyone else.
            </small>
          </div>
          <div
            class="mb-3"
          >
            <label
              class="form-label"
              for="formBasicPassword"
            >
              Password
            </label>
            <input
              class="form-control"
              id="formBasicPassword"
              placeholder="Password"
              type="password"
            />
          </div>
          <div
            class="container-login100-form-btn register-but"
          >
            <a
              href="/"
              name="dashboard"
              style="text-decoration: none; color: white; font-family: sans-serif;"
            >
              <button
                class="login100-form-btn"
                type="submit"
              >
                Login
              </button>
            </a>
          </div>
          <div
            class="text-center p-t-46 p-b-20"
          >
            <span
              class="txt2"
            >
              Haven't registered with us ?
            </span>
          </div>
          <div
            class="container-login100-form-btn register-but"
          >
            <a
              href="/register"
              style="text-decoration: none; color: white; font-family: sans-serif;"
            >
              <button
                class="login100-form-btn"
              >
                Register
              </button>
            </a>
          </div>
        </form>
        <div
          class="login100-more"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
